include/cookPackage.hpp

```diff
@@ -0,0 +1,19 @@
+/*
+** EPITECH PROJECT, 2024
+** Plaza
+** File description:
+** No file there , just an epitech header example .
+** You can even have multiple lines if you want !
+*/
+
+#pragma once
+#include <iostream>
+#include "Cooks.hpp"
+
+struct CookPackage {
+    Ticket *ticket;
+    std::size_t timeToCook;
+    Cooks *cooker;
+    std::vector<Ticket> *_doneCommandsList;
+    CookPackage(Ticket *ticket, std::size_t timeToCook, Cooks *cooker, std::vector<Ticket> *doneCommandsList) : ticket(ticket), timeToCook(timeToCook), cooker(cooker), _doneCommandsList(doneCommandsList) {}
```

Comment:
I don't think this constructor is useful for this struct, since it's the default comportement

Reply:
I can only make a default constructor, but i still need to fill the struct